scripts/apply_libsass_patches.R

```diff
@@ -4,12 +4,19 @@
 
 library(rprojroot)
 
-patch_dir <- rprojroot::find_package_root_file("scripts/patches")
+# Remove libsass test files (the entire `test` directory) since they contain unicode
+# directory paths, and R CMD check provides a warning.
+unlink(
+  find_package_root_file("src/libsass/test"),
+  recursive = TRUE
+)
```

Comment:
The equivalent thing used to be done via `scripts/patches/002-remove-libsass-tests.patch`, but that had conflicts, and it seems better to it this way instead